dex/src/main/scala/com/wavesplatform/dex/api/ws/protocol/WsAddressSubscribe.scala

```diff
@@ -15,9 +15,25 @@ import pdi.jwt.{JwtAlgorithm, JwtJson, JwtOptions}
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
-final case class WsAddressSubscribe(key: Address, authType: String, jwt: String, flags: Set[WsAddressFlag] = Set.empty) extends WsClientMessage {
+final case class WsAddressSubscribe(private val key: Address, authType: String, jwt: String, flags: Set[WsAddressFlag] = Set.empty)
+    extends WsClientMessage {
   override val tpe: String = WsAddressSubscribe.tpe
 
+  private def maybeWithDebug(payload: JwtPayload): JwtPayload =
```

Comment:
This should be `private def isDebug(payload: JwtPayload): Boolean`. See below

Reply:
fixed